modules/standard/Random.chpl

```diff
@@ -973,21 +977,56 @@ module Random {
                 element type, with no negative values and at least one non-zero
                 value. The domain must be equal to that of ``arr.domain``.
 
-     :return: An element chosen from ``arr`` is ``size == 1``, or an array of
+     :return: An element chosen from ``arr`` if ``size == 1``, or an array of
               element chosen from ``arr`` if ``size > 1`` or ``size`` is a
               domain.
 
      :throws IllegalArgumentError: if ``arr.size == 0``,
                                    if ``arr`` contains a negative value,
```

Comment:
Here's a pre-existing problem with the doc comment -- `arr` can have negative values, or be all 0.  It's `prob` that has these restrictions.

Reply:
Yeah! Thanks for pointing out. I will fix that :-)